iXBRLViewerPlugin/viewer/src/js/fact.js

```diff
@@ -247,3 +247,7 @@ Fact.prototype.escaped = function () {
 Fact.prototype.footnotes = function () {
     return $.map(this.f.fn || [], (fn, i) => this._report.getItemById(fn));
 }
+
+Fact.prototype.isHidden = function () {
+    return this._ixNode.wrapperNode.length == 0;
```

Comment:
Is it ok that -ix-hidden elements are reported as non-hidden?

Reply:
I think so.  From an end user perspective they behave like non-hidden facts.

If you're reviewing a document, then I think it might be desirable to be able to find -ix-hidden elements as there's slightly more scope for tagging errors, as they're not constrained by transformation rules.  Perhaps we could add an additional filter toggle, although I'm not sure what to call it.

Reply:
Works for me.  I also don't have words, I just wanted to be sure it was deliberate.